azuremetadata

```diff
@@ -52,15 +52,15 @@ try:
     # internal server error, that's the best thing we have to go on to
     # figure out whether or not we are in ASM.
     # ASM gets retired in 2023, rip this code out, it's ugly!
-    attested_data_url = 'http://169.254.169.254/metadata/attested/document'
+    attested_data_url = 'http://169.254.169.254/metadata/instance/compute'
```

Comment:
Does `/metadata/instance/compute` also return the attested data?

Reply:
No, `/metadata/instance/compute` does not return attested data

It returns the info in [sample 4 response](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service?tabs=linux#sample-4-get-more-information-about-the-vm-during-support-case) or raise a HTTP error if vm is `classic`

Reply:
OK, so there's a variable called `attested_data_url` which never returns any attested data (which we expect to have on RMT side for verification of non-classic VMs). Confused I am.